_data/mentors.yml

```
@@ -1026,4 +1026,33 @@
     extra: |
       Software Development Strategies, Resume Review, Preparation for The Technical Interview
   network:
-    - linkedin: https://www.linkedin.com/in/liliia-rafikova-635328104/
\ No newline at end of file
+    - linkedin: https://www.linkedin.com/in/liliia-rafikova-635328104/
+
+- name: Monique Grinstein
+  disabled: false
+  sort: 200
+  hours: 2
+  position: Software Engineer, The LEGO Group
+  type: ad-hoc
+  index: 33
+  location: London, UK
+  bio: |
+    I work mainly with Backend and AWS. I am keen to help people start their journey in Software Engineering and Cloud Computing. I can share some knowledge from the industry, help you review your CV to apply for a job, and anything else around that. If you’re looking to get AWS certified, I could also help you come up with a study plan, and share tips and study materials. I currently hold two certifications: Cloud Practitioner & Solutions Architect - Associate.
+  image: assets/images/mentors/monique_grinstein.jpeg
+  languages: English, Portuguese, French
+  availability: []
```

I guess she will be available next 2 months at least. 

```suggestion
  availability: [ 6 , 7 ]
```

---

@dricazenck what is this one for / how is it used? :)